docs/articles_en/assets/snippets/ov_custom_op.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# ! [op:common_include]
+import numpy as np
+from openvino import Op
+from openvino.runtime import DiscreteTypeInfo
+# ! [op:common_include]
+
+
+
+# ! [op:header]
+class Identity(Op):
+    class_type_info = DiscreteTypeInfo("Identity", "extension")
+
+    def get_type_info(self):
+        return Identity.class_type_info
+# ! [op:header]
+
+# ! [op:ctor]
+    def __init__(self, inputs, attrs=None):
+        super().__init__(self)
+        self.set_arguments(inputs)
+        self.constructor_validate_and_infer_types()
+        if attrs is not None:
+            self._attrs = attrs
```

Comment:
Does it mean that I can stuff any attributes to this op? Looks a bit artificial. Identity doesn't have any attributes, so it is not very good example for attributes demonstration.

Reply:
Yes, Identity doesn't have any attributes and if I remove them, I'm worried that in this case both C++ and Python snippets don't cover how to create operation with attributes.
This is the only reason why I added it

Reply:
This should be a reason to extend both C++ and Python examples to demonstrate attributes. It should be shown in a dedicated section just to highlight that you don't need attribute-related stuff if you don't have attributes, which is most likely true in case of Python Op as you can carry attributes in the op object itself and not to show them as ov::Op attributes. Showing them as attributes will be practically used for debugging purposes only at this moment: you cannot serialize the attributes to IR and then read them back for PythonOp (as we don't support PythonOp deserialization I suppose).

Reply:
removed attributes part